org.kde.WaylandDecoration.QWhiteSurGtkDecorations.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "org.kde.WaylandDecoration.QWhiteSurGtkDecorations",
+    "branch": "5.15-23.08",
+    "runtime": "org.kde.Platform",
+    "build-extension": true,
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.15-23.08",
+    "appstream-compose": false,
+    "separate-locales": false,
+    "modules": [
+        {
+            "name": "qwhitesurgtkdecorations",
+            "buildsystem": "cmake-ninja",
+            "config-opts": ["-DCMAKE_INSTALL_PREFIX=/usr/share/runtime/lib/plugins/QWhiteSurGtkDecorations",
```

Comment:
instead of setting `CMAKE_INSTALL_PREFIX` you should set a top level `build-options` with a `prefix`